README.md

```diff
@@ -9,6 +9,16 @@ Tools to manipulate DRUID trees and content directories
 
 ## Usage
 
+### with strict argument
```

Comment:
maybe add that this is the preferred mode going forward, and that the flag default is just for backwards compatibility?

Reply:
given that we commonly use vowel druids as test druids, and given that druid-tools is used by a ton of our code (see https://github.com/sul-dlss/druid-tools/network/dependents), I wasn't sure about this decision.  Perhaps we can post a question in dlss-infrastructure channel?

Deprecation isn't breaking stuff, to be sure, but the messages sure can get old.